checklists/avd_checklist.en.json

```diff
@@ -518,64 +302,496 @@
+            "guid": "bd362caa-ab79-4b19-adab-81932c9fc9d1",
+            "severity": "Low",
+            "link": "https://docs.microsoft.com/azure/virtual-desktop/app-attach-faq"
+        },
+        {
+            "category": "Compute",
+            "subcategory": "MSIX & AppAttach",
+            "text": "Disable auto-update for MSIX packages",
+            "description": "MSIX app attach doesn't support auto-update for MSIX applications, then should be disabled.",
+            "guid": "bb88037f-5e6b-4fbb-aed5-03547cc447e8",
+            "severity": "Low",
+            "link": "https://docs.microsoft.com/azure/virtual-desktop/app-attach-faq"
+        },
+        {
+            "category": "Compute",
+            "subcategory": "MSIX & AppAttach",
+            "text": "Review operating systems support",
+            "description": "In order to leverage MSIX & App Attach, guest OS image for AVD Host pool must be Windows 10/11 Enterprise or Windows 10/11 Enterprise Multi-session, version 2004 or later.",
+            "guid": "26128a71-f0f1-4cac-9d9e-f1d5e832e42e",
+            "severity": "Medium",
+            "link": "https://docs.microsoft.com/azure/virtual-desktop/app-attach-faq"
+        },
+        {
+            "category": "Compute",
+            "subcategory": "Session Host",
+            "text": "Evaluate the usage of Gen2 VM for Host Pool deployment",
+            "description": "Once selected the VM SKU that will be used for Host Pool deployment, it is recommended to use Gen2 type of the SKU for higher security and improved capabilities.",
+            "guid": "e4633254-3185-40a1-b120-bd563a1c8e9d",
+            "severity": "Medium",
+            "link": "https://docs.microsoft.com/en-us/azure/virtual-machines/generation-2"
+        },
+        {
+            "category": "Networking",
+            "subcategory": "Networking",
+            "text": "Determine if hybrid connectivity is required to connect to on-premises environment",
+            "description": "If required to connect to on-premises environment, assess the current connectivity option or plan for the required connectivity (Express Route, Azure S2S or 3rd-party NVA VPN). ",
+            "guid": "dd399cfd-7b28-4dc8-9555-6202bfe4563b",
+            "severity": "Medium",
+            "link": "https://docs.microsoft.com/azure/architecture/reference-architectures/hybrid-networking/"
+        },
+        {
+            "category": "Networking",
+            "subcategory": "Networking",
+            "text": "Determine Azure Virtual Network (VNet) placement for each AVD Host Pool",
+            "description": "AVD Host Pools can be deployed in either Azure Virtual WAN or traditional 'Hub & Spoke' network topologies. It is recommended to deploy each Host Pool in a separate 'spoke' VNet, using 'hub' is not recommended.",
```

Comment:
We can reference the zero trust article for this as well: https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/scenarios/wvd/eslz-network-topology-and-connectivity